### DISCUSSION TITLE
中英文标点问题

### DISCUSSION THREAD

**Discussion by Dreamerxic** [SYSU-SCC/sysu-thesis#100] [General]
请问模板在哪里确定打印中文还是英文标点？在tex文件的的英文标点到pdf就变成中文标点了。

**Comment by sikouhjw** [comment]
截图

**Comment by Dreamerxic** [comment]
![image](https://github.com/SYSU-SCC/sysu-thesis/assets/44291865/4bca33a4-4fff-4bca-806a-0b193c13e2d8)
![image](https://github.com/SYSU-SCC/sysu-thesis/assets/44291865/d699efce-0c88-43dc-aeec-227d8601d842)
仔细对比看中英文符号有一些微小的差别，但英文半角的引号是有问题的

**Reply by Dreamerxic** [reply]
@sikouhjw

**Reply by sikouhjw** [reply]
英文全角=中文全角吧，所以才一模一样。

我没理解你的问题在哪
